shared-libs/user-management/src/token-login.js

```diff
@@ -193,8 +194,7 @@ const generateTokenLoginDoc = (user, userSettings, token, appUrl) => {
       tasks: [],
     };
 
-    appUrl = (config.get('app_url') || appUrl).replace(/\/+$/, '');
-    const url = `${appUrl}/medic/login/token/${token}`;
+    const url = `${appUrl.replace(/\/+$/, '')}/medic/login/token/${token}`;
```

Comment:
FTR, I doulbe/triple checked to confirm that the `appUrl` being passed in here is populated from `config.get('app_url')`. So, the duplicate call here is unnecessary.